### ISSUE TITLE
PHP notice in Squiz.Commenting.FunctionComment when checking malformed @throws comment

### ISSUE THREAD

**Issue by aik099** [squizlabs/PHP_CodeSniffer#485]
When having DocBlock of function like this:
```php
/**
 * Test
 *
 * @return array
 * @throws E 
 */
```
Note space after `@throws E`, then "Squiz.Commenting.FunctionComment" sniff is triggering `Uninitialized string offset: 0` notice because in that case comment is text is matched via regexp and is an empty string. Then check for 1st char is a letter is invoked in 205 line and that's triggers a notice.

I think that `} else if ($comment === null) {` in line 187 in fact should be `} else if (empty($comment) === true) {` to catch both undefined and empty comments.

**Comment by gsherwood** [comment]
Pushed a fix. Thanks.

**Comment by aik099** [comment]
Thanks.
